client/my-sites/checkout/checkout-thank-you/redesign-v2/pages/content/get-default-footer-details.ts

```diff
@@ -0,0 +1,24 @@
+import { localizeUrl } from '@automattic/i18n-utils';
+import { translate } from 'i18n-calypso';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import type { ThankYouFooterDetailProps } from 'calypso/components/thank-you-v2/footer';
+
+export default function getDefaultFooterDetails( context: string ): ThankYouFooterDetailProps[] {
```

Comment:
This is the default support blurb when there isn't any other specific blurbs to surface in the congrats page.

Reply:
👍